server/utils/applications/withdrawables/sortWithdrawables.test.ts

```diff
@@ -6,6 +6,14 @@ import { sortWithdrawables } from './sortWithdrawables'
 describe('sortWithdrawables', () => {
```

Comment:
So we never want applications to be selectable?

Given the sorted withdrawables are currently always filtered, would it make more sense for the included types to be an argument to the method so you can do a single filter:

```ts
const sortAndFilterWithdrawables = (
  withdrawables: Array<Withdrawable>,
  typesToInclude: Array<Withdrawable['type']>
) => {
  return withdrawables.filter(withdrawable => typesToInclude.includes(withdrawable.type)
  ...
}
```

Otherwise you're filtering out applications, then sorting the remaining withdrawables, then filtering some more types out in the controller

Reply:
yeah this is a much better approach, I was going for something quick but it's better to try and improve readability if we can